domain_config_ui/src/Form/DeleteForm.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Drupal\domain_config_ui\Form;
+
+use Drupal\Component\Utility\SafeMarkup;
+use Drupal\Core\Url;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\domain_config_ui\Controller\DomainConfigUIController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+/**
+ * Class DeleteForm.
+ */
+class DeleteForm extends FormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'domain_config_ui_delete';
+  }
+
+  /**
+   * Build configuration form with metadata and values.
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, $config_name = NULL) {
+    if (empty($config_name)) {
+      $url = Url::fromRoute('domain_config_ui.list');
+      return new RedirectResponse($url->toString());
+    }
+
+    $elements = DomainConfigUIController::deriveElements($config_name);
+    $config = \Drupal::configFactory()->get($config_name)->getRawData();
+
+    $form['help'] = [
+      '#type' => 'item',
+      '#title' => SafeMarkup::checkPlain($config_name),
```

Comment:
This method is marked deprecated. As discussed in https://www.drupal.org/node/2549395
From the list of suggested replacements `Html::escape` seems closest to the intended behavior.

```suggestion
      '#title' => Html::escape($config_name),
```
This suggestion also needs an additional use statement: `use Drupal\Component\Utility\Html;` and the SafeMarkup use can be removed with this change.